src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Using the universal selector `*` for resetting margins and paddings can significantly impact performance and is against the guidelines. Please replace it with a more specific reset method.